app/controllers/summary_controller.rb

```diff
@@ -12,12 +12,12 @@ def new
   end
 
   def create
-    @summary = Summary.new(params.require(:summary).permit(:body))
+    @summary = Summary.new(params.require(:summary).permit(:body, :post_id))
     @post = Post.find(params[:post_id])
```

Comment:
I _could_ have simply used the `params[:post_id]` explicitly here in the controller, a la:

```ruby
@topic.post_id = params[:post_id]
```

That would have been ok too, IMO. Not sure which feels less like a hack.